Python/math/Lucky_Numbers.py

```diff
@@ -0,0 +1,49 @@
+'''
+Write a program to print all the Lucky Numbers upto the input number.
+Description:
+How to know if a number is lucky ?
+Start with a list of numbers beginning with 1:
+1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
+Consistently number (every much number) in the list is dispensed with, leaving just the odd numbers:
+1 3 5 7 9 11 13 15 17 19 21 23 25
+The main number excess in the list after 1 is 3, so every third number which stays in the list (only one out of every odd various of 3) is dispensed with. The first of 
+these is 5:
+1 3 7 9 13 15 19 21 25
+The following enduring number is presently 7, so every seventh leftover number is disposed of. The first of these is 19:
+1 3 7 9 13 15 21 25
+Keep eliminating the nth leftover numbers, where n is the following number in the list after the last surviving number. Next in this model is 9. Keep repeating until the sieve
+of elimination becomes larger than the index of the number[ which is to be checked if lucky ] , then the number is called Lucky Number. If a number is eliminated by any seieve 
+then the number is not a lucky number.

```

Comment:
Please make it short and readable

Reply:
@ankitaggarwal23 I tried to make it more readable , but really cannot make it any shorter ,as making shorter may hamper the clarity of the problem.